"Get the Code" in Cheat Sheet fixed.Issue solettaproject/soletta-dev-app#62

kamidipreetham opened the pull request:
Fixed copy to clipboard issue in Cheat sheet page.

bdilly commented on doc/node-types-html/js/page/ExamplesView.js:
```diff
@@ -1,199 +1,200 @@
-    window.ExamplesView = ExamplesView;
-
-}(window));
+//Commenting out as modified version of this script is loaded with the page inline.
```
What is the point on putting it inline?
I would suggest to keep it in a separate file

But if we have a good reason to have it inline, you should remove previous file, don't comment it please

bdilly replied:
@kamidipreetham  Also when such kind of change is required (moving code for another file) it's highly indicated to make a commit moving the file without any changes, and another commit just making changes on code itself, otherwise it's really hard to review it.

kamidipreetham replied:
@bdilly execCommand API which is required for the 'copy' operation works only when the script is loaded inline. I'm sorry for the trouble caused due to the commenting part, will make the necessary changes now.

brunobottazzini commented on doc/node-types-html/index.html.in:
```diff
@@ -19,6 +19,216 @@
+            y=copycode();
+            console.log(y);
+            $(this).fadeOut(100,showCopiedFeedback);
+    } 
```
Missing indentation here

brunobottazzini commented:
Okay I got why the JS code should be inline but, I'd like you to be more specific in your commit message explaining why we are doing this kind of move.

brunobottazzini commented:
Instead of just putting "Fixed issue solettaproject/soletta-dev-app#62" in the commit message, please try making an introductory message of what issue is being fixed adding a little bit of more details.

brunobottazzini commented on doc/node-types-html/index.html.in:
```diff
@@ -19,6 +19,216 @@
+            return copycheck;
+        }
+        function showCopiedFeedback(){
+
```
You can remove this extra line in the code :)

brunobottazzini commented:
@kamidipreetham following our contribution flow. 

Some hint here:
Close this PR and open Another one with a V2.  There, you will put all the details of the differences from this PR.

Example of a V2:
https://github.com/solettaproject/soletta/pull/1840

Some details of how to contribute with us:
https://github.com/solettaproject/soletta/wiki/Contributing

Thanks :+1:

brunobottazzini commented on doc/node-types-html/index.html.in:
```diff
@@ -19,6 +19,216 @@
+
+        /* load the contents of the selected entry */
+        function loadExampleContents(_data){
+
```
You can remove this empty line

brunobottazzini commented on doc/node-types-html/index.html.in:
```diff
@@ -19,6 +19,216 @@
+         * closes the view and restore the main content */
+        function close(){
+            el.fadeOut(250,onCloseComplete);
+        }
```
Put an extra line between functions

brunobottazzini commented on doc/node-types-html/index.html.in:
```diff
@@ -19,6 +19,216 @@
+            $('#example').hide();
+            el.trigger('examples:closed');
+        }
+        ExamplesView.prototype.show = function(_data){
```
put an extra empty line here

brunobottazzini commented on doc/node-types-html/index.html.in:
```diff
@@ -19,6 +19,216 @@
 
```
Since you are removing ExamplesView.js. you should remove references that point to it otherwise you will receive errors.

kamidipreetham commented:
Changes since V1:

- Added commits [83febb2](https://github.com/kamidipreetham/soletta/commit/83febb2a1d931e4e119ce6fc88c50c685e652be4#diff-e7801d701df50633c3a85baa614c0000)  and [424af46](https://github.com/kamidipreetham/soletta/commit/424af46186c7af83150c435a3647884b95a30270#diff-e7801d701df50633c3a85baa614c0000)
- Removed all references of ExamplesView.js.(code of ExamplesView.js is now inline)
- Proper code formatting.
